wgpu-hal/src/dx11/adapter.rs

```diff
@@ -14,6 +14,7 @@ impl crate::Adapter<super::Api> for super::Adapter {
     unsafe fn texture_format_capabilities(
         &self,
         format: wgt::TextureFormat,
+        capabilities: &crate::Capabilities,
```

Comment:
I don't think this is the best way to go about this. We should pass along what we need internal to the backends.

Reply:
Done